### DISCUSSION TITLE
Collapsible Section/ Accordion

### DISCUSSION THREAD

**Discussion by EbramTawfik** [microsoft/fluentui#27168] [Q&A] [answered]
I am trying to implement this design:

![image](https://user-images.githubusercontent.com/12788360/224451657-0d8360e4-db19-4645-b2ef-ba45ba2c0a22.png)

It's a Persona with a Collapsible section to assign a role.

I tried Nav and GroupedLists as recommended [here ](https://stackoverflow.com/questions/66005985/how-to-use-a-collapsible-list-in-fluent-ui-react) but this didn't work for me as the children of the  Collapsible  are not links. Any recommendation about how to implement Collapsible section or Accordion for Fluent UI?

**Comment by behowell** [comment, marked as answer]
Hi @EbramTawfik, you could use an [Accordion](https://react.fluentui.dev/?path=/docs/components-accordion--default), with a [Persona](https://react.fluentui.dev/?path=/docs/components-persona--default) in the AccordionHeader, and a [RadioGroup](https://react.fluentui.dev/?path=/docs/components-radiogroup--default) in the AccordionPanel.

For example:
```jsx
<Accordion multiple>
  <AccordionItem value="1">
    <AccordionHeader>
      <Persona ... />
    </AccordionHeader>
    <AccordionPanel>
      <RadioGroup>
        <Radio label="User" />
        <Radio label="Owner" />
      </RadioGroup>
    </AccordionPanel>
  </AccordionItem>
  <!-- add more AccordionItems -->
</Accordion
```
